dts/riscv/nordic/nrf54h20_cpuflpr.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54h20.dtsi>
+
+cpu: &cpuflpr {};
+clic: &cpuflpr_clic {};
+cpuppr_vevif: &cpusys_vevif_tx {};
+cpuflpr_vevif: &cpuflpr_vevif_rx {};
+cpusys_vevif: &cpusys_vevif_tx {};
+
+/delete-node/ &cpuapp;
+/delete-node/ &cpuapp_peripherals;
+/delete-node/ &cpuapp_ppb;
+/delete-node/ &cpuapp_ram0;
+/delete-node/ &cpurad;
+/delete-node/ &cpurad_peripherals;
+/delete-node/ &cpurad_ppb;
+/delete-node/ &cpurad_ram0;
+/delete-node/ &cpuppr;
```

Comment:
Wouldn't you be better off reworking `nordic/nrf54h20.dtsi` into multiple files? like a "common" one and others for the specific domain. It's a bit odd to have so many delete-node

Reply:
Base file has been designed as a pseudo-system-devicetree.